### PR TITLE
Update `dev` Version

### PR DESCRIPTION
**Pull request by michaelweinold**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by michaelweinold** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,8 +28,7 @@ requirements:
     - numpy
     - peewee >=3.9.4
     - platformdirs
-    - python >=3.5
-    - requests >=1.1.0
+    - python >=3.9
```
Comment (cmutel) [inline recipe/meta.yaml]: Need to add `bw-projects` as well.

**Comment by michaelweinold** [comment]
😑 I so don't need [this](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=768476&view=logs&j=516f237f-98f1-50a4-0946-c43fe5437b81&t=19870002-9ea0-5c64-a2c3-e2872fb3f101):

```
ValueError: License file given in about/license_file (/home/conda/recipe_root/LICENSE.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)
##[error]Bash exited with code '1'.
```

I never touched the license file - nor the variable that points to it. Ugh.

File: recipe/meta.yaml (hunk 1 above)
Comment (michaelweinold) [reply recipe/meta.yaml]: ...and to make double-sure, that's bw-projects, but bw_processing?

Comment (cmutel) [reply recipe/meta.yaml]: And it should be `bw-projects >=2.1.0`

**Comment by michaelweinold** [comment]
...closing after discussion with @cmutel. Since `bw-projects` is not yet on conda-forge, we'll instead release 4.0.dev21.1.

**Pull request closed without merging**